### PR TITLE
Added testing framework & fixed coding style errors

### PR DESCRIPTION
**Pull request by RaenonX**
Already based on the most main branch update 001deed.

### CODE REVIEW

**Review by Jmays2 — CHANGES_REQUESTED** [review]
Remove anything that still uses SKU.

File: app/controllers/staff_controller.py [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
-
-    def make_outStock(self, prod_id):
+    @staticmethod
+    def make_out_stock(prod_id):
+        """Doc."""
+        # FIXME: Add documentation
         SKU = Stripe.SKU.list(product=prod_id)
-        updated = Stripe.SKU.modify(
-            SKU["data"][0]["id"],
+        Stripe.SKU.modify(
```
Comment (Jmays2) [inline app/controllers/staff_controller.py]: Remove

File: app/controllers/staff_controller.py [hunk 2]
```diff
@@ -1,31 +1,44 @@
...
+            SKU['data'][0]['id'],
```
Comment (Jmays2) [inline app/controllers/staff_controller.py]: Remove

File: app/controllers/staff_controller.py [hunk 3]
```diff
@@ -1,31 +1,44 @@
...
         )
         return updated_product
 
-
-    def make_outStock(self, prod_id):
+    @staticmethod
+    def make_out_stock(prod_id):
+        """Doc."""
+        # FIXME: Add documentation
         SKU = Stripe.SKU.list(product=prod_id)
```
Comment (Jmays2) [inline app/controllers/staff_controller.py]: Remove

File: app/controllers/staff_controller.py [hunk 4]
```diff
@@ -1,31 +1,44 @@
...
 
 class StaffController:
+    """Controller for staff."""
 
-    def make_inStock(self, prod_id):
+    @staticmethod
+    def make_in_stock(prod_id):
+        """Doc."""
+        # FIXME: Add documentation
         SKU = Stripe.SKU.list(product=prod_id)
```
Comment (Jmays2) [inline app/controllers/staff_controller.py]: Remove

File: app/controllers/staff_controller.py [hunk 5]
```diff
@@ -1,31 +1,44 @@
...
-        updated = Stripe.SKU.modify(
-            SKU["data"][0]["id"],
+        Stripe.SKU.modify(
```
Comment (Jmays2) [inline app/controllers/staff_controller.py]: Remove

File: app/controllers/staff_controller.py (hunk 4 above)
Comment (Jmays2) [inline app/controllers/staff_controller.py]: Remove

File: app/controllers/order_controller.py [hunk 6]
```diff
@@ -1,52 +1,65 @@
...
-            updated_product = Stripe.Product.modify(
+            Stripe.Product.modify(
                 prod_id,
                 active=True,
-            )
+            )  # updated_product
         else:
             updated = Stripe.SKU.modify(
-                SKU["data"][0]["id"],
+                SKU['data'][0]['id'],
```
Comment (Jmays2) [inline app/controllers/order_controller.py]: Remove.

File: app/controllers/order_controller.py [hunk 7]
```diff
@@ -1,52 +1,65 @@
...
+        # removes amount of item of given product ID from the inventory
+        # negative amounts will add to inventory
         SKU = Stripe.SKU.list(product=prod_id)
-        inventory = SKU["data"][0]["inventory"]["quantity"]-amount
-        #Precursor for out of stock warning
-        if (inventory > 0):
+        inventory = SKU['data'][0]['inventory']['quantity'] - amount
+        # Precursor for out of stock warning
+        if inventory > 0:
             updated = Stripe.SKU.modify(
```
Comment (Jmays2) [inline app/controllers/order_controller.py]: Remove

File: app/controllers/order_controller.py [hunk 8]
```diff
@@ -1,52 +1,65 @@
...
-                SKU["data"][0]["id"],
+                SKU['data'][0]['id'],
```
Comment (Jmays2) [inline app/controllers/order_controller.py]: Remove

File: app/controllers/order_controller.py [hunk 9]
```diff
@@ -1,52 +1,65 @@
...
-#negative amounts will add to inventory
-    def removeFromInventory(self, prod_id, amount):
+    @staticmethod
+    def remove_from_inventory(prod_id, amount):
+        """Doc."""
+        # FIXME: Seems like a testing method, either convert to a working code and document it or delete it after test
+        #   Also some unused variables were commented to pass CI
+        # removes amount of item of given product ID from the inventory
+        # negative amounts will add to inventory
         SKU = Stripe.SKU.list(product=prod_id)
```
Comment (Jmays2) [inline app/controllers/order_controller.py]: Remove

**Review by Jmays2 — CHANGES_REQUESTED** [review]

**Review by Jmays2 — APPROVED** [review]

**Review by tmehmen2 — APPROVED** [review]

**Pull request merged**